vendor/golang.org/x/sys/unix/ioctl_zos.go

```diff
@@ -51,13 +49,13 @@ func IoctlSetTermios(fd int, req uint, value *Termios) error {
 // for those, IoctlRetInt should be used instead of this function.
 func IoctlGetInt(fd int, req uint) (int, error) {
 	var value int
-	err := ioctl(fd, req, uintptr(unsafe.Pointer(&value)))
+	err := ioctlPtr(fd, req, unsafe.Pointer(&value))
 	return value, err
 }
 
 func IoctlGetWinsize(fd int, req uint) (*Winsize, error) {
 	var value Winsize
-	err := ioctl(fd, req, uintptr(unsafe.Pointer(&value)))
+	err := ioctlPtr(fd, req, unsafe.Pointer(&value))
 	return &value, err
 }
 
```

Comment:
The code review exists over two functions `IoctlSetWinsize` and `IoctlGetInt` as follows:
 
`IoctlSetWinsize(fd int, req uint, value *Winsize) error`:

- The TODO comment suggests hardcoded modification.
- The code patch replaces the use of `ioctl()` with `ioctlPtr()`.
- There's no need for `runtime.KeepAlive(value)` anymore.
- `err` is returned directly to the caller.

Suggestion: Given that `TIOCSWINSZ` probably isn't going away anytime soon, it might be reasonable to hardcode it instead of using a parameter.

`IoctlGetInt(fd int, req uint) (int, error)`:

- The usage of `ioctl()` with a pointer to an integer has been replaced by `ioctlPtr()`.
- Value is created in a better standard way of initializing variables.

Suggestion: The changes are adequate; nothing further needs to be addressed unless there was some reason not to use `ioctlPtr()`.